diff --git a/lib/models/monodetr/ops/modules/ms_deform_attn.py b/lib/models/monodetr/ops/modules/ms_deform_attn.py
--- a/lib/models/monodetr/ops/modules/ms_deform_attn.py
+++ b/lib/models/monodetr/ops/modules/ms_deform_attn.py
@@ -51,10 +51,7 @@ from torch.nn.modules.utils import _single, _pair, _triple, _list_with_default
 from torch.nn import grad
 from torch import _VF
 from torch._jit_internal import boolean_dispatch, List, Optional, _overload, Tuple
-if float(torch.__version__.split('.')[0]) == 0 or (float(torch.__version__.split('.')[0]) == 1 and float(torch.__version__.split('.')[1])) < 7:
-    from torch._overrides import has_torch_function, handle_torch_function
-else:
-    from torch.overrides import has_torch_function, handle_torch_function
+from torch.overrides import has_torch_function, handle_torch_function
 Tensor = torch.Tensor
 
 from torch.nn.functional import linear, pad, softmax, dropout
